Cargo.toml

```diff
@@ -13,7 +13,12 @@ name = "glow"
 path = "src/lib.rs"
 
 [target.'cfg(not(any(target_arch = "wasm32")))'.dependencies]
-glutin = "0.19"
+glutin = { version = "0.19", optional = true }
+sdl2 = { version = "0.32", optional = true }
+
+[features]
+default = ["glutin"]
```

Comment:
I think we can move `glutin` out of default features too, and just use `--features=glutin`